src/components/ugas/GasStats.vue

```diff
@@ -441,7 +441,7 @@ ul {
   align-items: baseline;
   font-size: 0.875rem;
   line-height: 1.25rem;
-  font-weight: 600;
+  font-weight: 600px;
```

Comment:
font weight is never in pixels
```suggestion
  font-weight: 600;
```

Reply:
I did this because this somehow fixed a safari bug - but have now undone it.

Reply:
I did this because this fixed a safari bug - but have now undone it.